manager.spec.yaml

```diff
@@ -1035,6 +1035,11 @@ components:
                         description: Date of last evaluation.
                         example: '2018-09-22T16:00:00+00:00'
                         nullable: true
+                      rule_id:
```

Comment:
@tkasparek is it `string` or `array`?

Reply:
String, see the line below. It's just identifier of the rule like CVE_123|bad_kernel